todo-app/src/todo.json

```diff
@@ -0,0 +1,20 @@
+[
+ {
+          "id": "1",
+          "description": "Wake up",
+          "deadline": "Wed Sep 13 2017",
+          "done":"true"
```

Comment:
Check out the [JSON data types](https://www.w3schools.com/js/js_json_datatypes.asp). You need to be using booleans here, not `"true"` as a string - this is getting into very dangerous territory for things like equality comparison etc.